### ISSUE TITLE
get table list error when click package is 8.0+

### ISSUE THREAD

**Issue by luyinhua** [techouse/mysql-to-sqlite3#15]
**Describe the bug**
get table list error when click package is 8.0+

**Expected behaviour**
when i run mysql-to-sqlite3 , but table list is some wrong .

**Actual result**
install the package click when 7.1.2

**System Information**

```bash
$ mysql2sqlite --version
```

**Comment by luyinhua** [comment]
python 3.6.9 
click 8.0.1 
mysql2sqlite 1.4.4 

no work

**Comment by techouse** [comment]
Could you please use the bug report template and supply the full output of `mysql2sqlite --version`?

**Comment by luyinhua** [comment]
| software               | version                                               |
|------------------------|-------------------------------------------------------|
| mysql-to-sqlite3       | 1.4.4                                                 |
|                        |                                                       |
| Operating System       | Darwin 19.6.0                                         |
| Python                 | CPython 3.6.9                                         |
| MySQL                  | mysql  Ver 8.0.26 for macos10.15 on x86_64 (Homebrew) |
| SQLite                 | 3.28.0                                                |
|                        |                                                       |
| click                  | 8.0.1                                                 |
| mysql-connector-python | 8.0.25                                                |
| python-slugify         | 5.0.2                                                 |
| pytimeparse            | 1.1.8                                                 |
| simplejson             | 3.17.2                                                |
| six                    | 1.15.0                                                |
| tabulate               | 0.8.9                                                 |
| tqdm                   | 4.61.1                                                |

**Comment by techouse** [comment]
[This test passed](https://github.com/techouse/mysql-to-sqlite3/runs/3269201553?check_suite_focus=true#step:6:146) so I'm not sure what kind of problem you have.

Can you supply a bit more info that just

> no work

**Comment by luyinhua** [comment]
Connected to pydev debugger (build 202.7660.27)
file /Users/apple/workspace/QualityControl/backend/grouper/grouper_data/drg_default_7e89bdb20350fd10ad2bb9da42c09299.sqlite is build,waiting some time...
bash /Users/apple/workspace/QualityControl/backend/grouper/./init_sqlite_db.sh  -h 172.xxxx114 -n xxx -u xxxx -p xxxxx -k 3306 -l drg_adrg,drg_adrg_drg,drg_adrg_icd,drg_cc,drg_mcc,drg_exclusion,drg_mdc_icd,drg_mdc,drg_icd,drg_weight,md_icd10,md_icd10_zl,md_icd9,code_mapping,fc_rule_package,fc_rule_package_operation,fc_rule_definition,fc_rule_definition_index,fc_qc_operation -o /Users/apple/workspace/QualityControl/backend/grouper/grouper_data/drg_default_7e89bdb20350fd10ad2bb9da42c09299.sqlite
开始导入：
2021-08-07 20:59:34 INFO     Done!
file /Users/apple/workspace/QualityControl/backend/grouper/grouper_data/drg_lf_1aa2924a15d8d768144f1c660c0c132f.sqlite is build,waiting some time...
bash /Users/apple/workspace/QualityControl/backend/grouper/./init_sqlite_db.sh  -h 172.1xxxxx -n qc_drg_linfen -u xxx -p xxxxx -k 3306 -l drg_adrg,drg_adrg_drg,drg_adrg_icd,drg_cc,drg_mcc,drg_exclusion,drg_mdc_icd,drg_mdc,drg_icd,drg_weight,md_icd10,md_icd10_zl,md_icd9,code_mapping,fc_rule_package,fc_rule_package_operation,fc_rule_definition,fc_rule_definition_index,fc_qc_operation -o /Users/apple/workspace/QualityControl/backend/grouper/grouper_data/drg_lf_1aa2924a15d8d768144f1c660c0c132f.sqlite
开始导入：
2021-08-07 20:59:36 INFO     Done!

**Comment by techouse** [comment]
Seems to me you're using the CLI options incorrectly.

Check the readme again.

**Comment by luyinhua** [comment]
> Seems to me you're using the CLI options incorrectly.
> 
> Check the readme again.

but when i install the click==7.1.2 , it work all ok. i don't change other's

**Comment by techouse** [comment]
Hmm. 🧐 Interesting. Does it work without the `-t` option?

**Comment by luyinhua** [comment]
> Hmm. 🧐 Interesting. Does it work without the `-t` option?

yes, without the `-t` option, it work .

**Comment by techouse** [comment]
Ok, now I at least know where the issue is. I'll rectify it, but meanwhile, I'll limit Click to 7.x

**Comment by luyinhua** [comment]
> Ok, now I at least know where the issue is. I'll rectify it, but meanwhile, I'll limit Click to 7.x

**Comment by techouse** [comment]
Hotfixed in [v1.4.6](https://github.com/techouse/mysql-to-sqlite3/releases/tag/v1.4.6)

**Comment by techouse** [comment]
@luyinhua Can you please provide some more detail on exactly what breaks for you? 

I've just run the app using Click 8.0.1 on Python 3.9 and it works as expected.

What's more, is that I've got tests [here](https://github.com/techouse/mysql-to-sqlite3/blob/v1.4.6/tests/func/mysql_to_sqlite3_test.py#L676) and [here](https://github.com/techouse/mysql-to-sqlite3/blob/v1.4.6/tests/func/test_cli.py#L357) set up for this specific scenario and they all pass using Click 8.x, just as I've noted above, so I'm not sure what exactly is broken?

**Comment by techouse** [comment]
I've done some digging and identified that 

with `Click` v7.x I get the parameters as a `tuple` 
```bash
pip freeze | grep -i click
click==7.1.2
mysql2sqlite -f out.db -d test_db -u root -p -t content_files content_media content_usages iframes
<class 'tuple'>
('content_files', 'content_media', 'content_usages', 'iframes')
```
but with `Click` v8.x I get a `string`
```bash
pip freeze | grep -i click
click==8.0.1
mysql2sqlite -f out.db -d test_db -u root -p -t content_files content_media content_usages iframes
<class 'str'>
('content_files', 'content_media', 'content_usages', 'iframes')
```

**Comment by techouse** [comment]
Fixed properly in [v1.4.7](https://github.com/techouse/mysql-to-sqlite3/releases/tag/v1.4.7)
